app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -34,7 +34,7 @@ $light-gray: #ccc
 
 $body-background:         white !default
 $body-font-family:        'Lato', 'Lucida Grande', Helvetica, Arial, sans-serif !default
-$body-font-color:         #555 !default
+$body-font-color:         #333 !default
```

Comment:
You can now safely remove the color rule from:

````CSS
.inplace-edit--errors {
    line-height: 1.2em;
    color: #444;
}
````

The color was only added there to provide correct accessibility on red background.